src/quantuminspire/projectq/backend_qx.py

```diff
@@ -483,7 +576,7 @@ def _execute_cqasm(self) -> None:
         self._quantum_inspire_result = self._quantum_inspire_api.execute_qasm(
             self._cqasm,
             number_of_shots=self._num_runs,
-            backend_type=self._backend_type,
+            backend_type=self._backend,
```

Comment:
why did you name this backend? It should be a backend_type.

Reply:
I changed it back